### ISSUE TITLE
[7.7.x] Server crashes with "Total world volume is larger than the max supported size"

### ISSUE THREAD

**Issue by MrX13415** [StrangeLoopGames/EcoIssues#9261]
The server crashes with the following exception on 7.7.0 or 7.7.1 using the attached world.
Using version 7.6.3 and everything works fine.

World Download: [Broken Eco World](https://dl.icelane.net/share/ace4a2e8-978c-473d-848c-9aab48c177c9/Eco%20(Game)_Storage.zip)

```
Unbehandelte Ausnahme: System.AggregateException: Mindestens ein Fehler ist aufgetreten. ---> System.ArgumentException: Total world volume of 720 x * 4142,52246720679 y * 720 z = 2147483647 is larger than the max supported size of int.MaxValue (2147483647).
   bei Eco.Shared.Math.WorldPosition3i.Initialize(Vector3i worldSize)
   bei Eco.Shared.Voxel.ChunkGrid`1.AddChunk(TChunk chunk)
   bei Eco.Shared.Voxel.ChunkGrid`1.GetOrAddChunk(Vector3i chunkPos)
   bei Eco.World.WorldChunkGrid.SetBlock(Type blockType, Vector3i worldPos, Object[] args)
   bei Eco.Gameplay.Objects.WorldObject.PlaceWorldObjectBlocks()
   bei Eco.Gameplay.Objects.WorldObject.BeginInitialize()
   bei Eco.Gameplay.Objects.WorldObjectManager.<>c__DisplayClass14_0.<Eco.Core.Plugins.Interfaces.IInitializablePlugin.Initialize>b__0(KeyValuePair`2 pair)
   bei System.Threading.Tasks.Parallel.<>c__DisplayClass42_0`2.<PartitionerForEachWorker>b__1()
   bei System.Threading.Tasks.Task.InnerInvokeWithArg(Task childTask)
   bei System.Threading.Tasks.Task.<>c__DisplayClass176_0.<ExecuteSelfReplicating>b__0(Object )
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   bei System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   bei System.Threading.Tasks.Parallel.PartitionerForEachWorker[TSource,TLocal](Partitioner`1 source, ParallelOptions parallelOptions, Action`1 simpleBody, Action`2 bodyWithState, Action`3 bodyWithStateAndIndex, Func`4 bodyWithStateAndLocal, Func`5 bodyWithEverything, Func`1 localInit, Action`1 localFinally)
   bei System.Threading.Tasks.Parallel.ForEachWorker[TSource,TLocal](IEnumerable`1 source, ParallelOptions parallelOptions, Action`1 body, Action`2 bodyWithState, Action`3 bodyWithStateAndIndex, Func`4 bodyWithStateAndLocal, Func`5 bodyWithEverything, Func`1 localInit, Action`1 localFinally)
   bei System.Threading.Tasks.Parallel.ForEach[TSource](IEnumerable`1 source, Action`1 body)
   bei Eco.Gameplay.Objects.WorldObjectManager.Eco.Core.Plugins.Interfaces.IInitializablePlugin.Initialize(TimedTask timer)
   bei Eco.Server.PluginManager.InitializePlugins()
   bei Eco.Server.PluginManager..ctor()
   bei Eco.Server.Startup.Start(String[] args)
   bei Eco.Server.MainClass.Main(String[] args)

```

**Comment by pamduncan** [comment]
What world dimensions did you use?  Is it dividable by 4 and no more then 400X400?

**Comment by MrX13415** [comment]
The dimensions settings has never been touched. It is set to  72x72. 
An older version of this map, from a few eco versions ago works fine.

**Comment by SeanAtr0n** [comment]
should be fixed in 7.7.2
